localstripe/resources.py

```diff
@@ -478,27 +478,31 @@ def refunded(self):
     @classmethod
     def _api_list_all(cls, url, customer=None, created=None, limit=10):
         try:
-            assert _type(customer) is str and customer.startswith('cus_')
-            assert type(created) in (dict, str)
-            if type(created) is dict:
-                assert len(created.keys()) == 1 and \
-                    list(created.keys())[0] in ('gt', 'gte', 'lt', 'lte')
-                date = try_convert_to_int(list(created.values())[0])
-            elif type(created) is str:
-                date = try_convert_to_int(created)
-            assert type(date) is int and date > 1500000000
+            if customer:
```

Comment:
To be consistent with the rest of the code, can you use `is not None` (just here, not needed after `except AssertionError:`)?

Otherwise the API would accept values like `""` and `0`.

Reply:
Ok indeed good idea!
Here is the diff:
```diff
--- a/localstripe/resources.py
+++ b/localstripe/resources.py
@@ -478,9 +478,9 @@ class Charge(StripeObject):
     @classmethod
     def _api_list_all(cls, url, customer=None, created=None, limit=10):
         try:
-            if customer:
+            if customer is not None:
                 assert _type(customer) is str and customer.startswith('cus_')
-            if created:
+            if created is not None:
                 assert type(created) in (dict, str)
                 if type(created) is dict:
                     assert len(created.keys()) == 1 and \